typescript/ccip-server/package.json

```diff
@@ -0,0 +1,37 @@
+{
```

Comment:
Is the intention to publish this with our npm packages? If not, please add a `private:true` field here

Reply:
I don't know the answer to this. When should a package be included to be published?

Reply:
If we intend to have it published to NPM for other devs to consume then we would publish it. I'm not sure either so please add `private: true` for now and we'll remove it if we decide to publish

Reply:
Fixed.